### PR TITLE
docs: Get all users include/exclude nested data

### PR DESCRIPTION
**Pull request by tkrisztian95**
We introduced features for the User API get all users endpoint. These additional query params `exludeProperties` and `includeCustomField` can help optimize requests.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
 
 The example response includes the pagination URLs and query parameters in the `links` object so you know what to expect. 
 
+**Dynamically include/exclude nested data**
+
+If you're managing a large number of users, you can benefit from the query params `includeCustomField` and `excludeProperties`.
+
+By default all custom field is returned nested in the user model.
```
Comment (mrnyax) [inline apiary.apib:1425]:
```suggestion
By default all custom fields are returned nested in the user model.
```

File: apiary.apib [hunk 2]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
 
 The example response includes the pagination URLs and query parameters in the `links` object so you know what to expect. 
 
+**Dynamically include/exclude nested data**
+
+If you're managing a large number of users, you can benefit from the query params `includeCustomField` and `excludeProperties`.
+
+By default all custom field is returned nested in the user model. 
+Using the `includeCustomField` can help reduce the data transferred and fetch the only custom fields that are needed.
```
Comment (mrnyax) [inline apiary.apib:1426]:
```suggestion
Using the `includeCustomField` can help reduce the data transferred and only fetch custom fields that are needed. 
```

**Review by mrnyax — APPROVED** [review]
Just some grammar changes. I think Madeline is an expert at this.

File: apiary.apib [hunk 3]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
 
 The example response includes the pagination URLs and query parameters in the `links` object so you know what to expect. 
 
+**Dynamically include/exclude nested data**
+
+If you're managing a large number of users, you can benefit from the query params `includeCustomField` and `excludeProperties`.
```
Comment (madeline-brennan) [inline apiary.apib:1423]:
```suggestion
If you manage a large number of users, you can use the query params `includeCustomField` and `excludeProperties` to filter nested data.
```

File: apiary.apib [hunk 4]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
...
+
+By default all custom fields are returned nested in the user model.
```
Comment (madeline-brennan) [inline apiary.apib:1425]:
```suggestion
By default, all custom fields are nested in the user model.
```

File: apiary.apib [hunk 5]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
...
+Using the `includeCustomField` can help reduce the data transferred and only fetch custom fields that are needed.
```
Comment (madeline-brennan) [inline apiary.apib:1426]:
```suggestion
You can use the parameter `includeCustomField` to limit the returned fields to those that match the key passed in the param value.
```

File: apiary.apib [hunk 6]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
 
 The example response includes the pagination URLs and query parameters in the `links` object so you know what to expect. 
 
+**Dynamically include/exclude nested data**
+
+If you're managing a large number of users, you can benefit from the query params `includeCustomField` and `excludeProperties`.
+
+By default all custom fields are returned nested in the user model.
+Using the `includeCustomField` can help reduce the data transferred and only fetch custom fields that are needed. 
+When specified, the custom fields returned are limited to those matching the key passed in the param value.
```
Comment (madeline-brennan) [inline apiary.apib:1427]: This is helpful if you want to fetch a specific set of custom fields to reduce data transfer.

File: apiary.apib [hunk 7]
```diff
@@ -1416,6 +1418,15 @@ You then decide to delete 10 users before you call the API to fetch page 3. Now
...
+To include more than one simply append N times the `&includeCustomField={Key}` to the request.
```
Comment (madeline-brennan) [inline apiary.apib:1428]:
```suggestion
To include more than one custom field, add `&includeCustomField={Key}` to the request N times.
```

Comment (madeline-brennan) [reply apiary.apib:1428]: I'm not clear if you have to reenter the parameter every time or if you just add another {Key}. Can you clarify? Also, should {Key} be upper or lower case? it looks like it's lower case in line 1345

**Review by madeline-brennan — CHANGES_REQUESTED** [review]

File: apiary.apib [hunk 8]
```diff
@@ -1423,7 +1423,7 @@ The example response includes the pagination URLs and query parameters in the `l
 If you manage a large number of users, you can use the query params `includeCustomField` and `excludeProperties` to filter nested data.
 
 By default, all custom fields are nested in the user model.
-Using the `includeCustomField` can help reduce the data transferred and only fetch custom fields that are needed. 
+You can use the parameter `includeCustomField` to limit the returned fields to those that match the key passed in the param value.
 When specified, the custom fields returned are limited to those matching the key passed in the param value.
```
Comment (madeline-brennan) [inline apiary.apib:1427]:
```suggestion
This is helpful if you want to fetch a specific set of custom fields to reduce data transfer.
```

File: apiary.apib (hunk 7 above)
Comment (tkrisztian95) [reply apiary.apib]: You need to append the whole `includeCustomField={Key}` with another key as query param. The {Key} should be replaced with value like "Department" its just a palecholder. 

For example `/api/v1/user?includeCustomField=Department&includeCustomField=Email&...`

**Review by madeline-brennan — APPROVED** [review]

**Pull request merged**